src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -33,8 +33,16 @@ public static void BufferSize_GettingSameAsWindowSize()
     [Fact]
     public static void WindowWidth_WindowHeight_InvalidSize()
     {
-        Assert.Throws<ArgumentOutOfRangeException>("width", () => Console.WindowWidth = 0);
-        Assert.Throws<ArgumentOutOfRangeException>("height", () => Console.WindowHeight = 0);
+        if (Console.IsOutputRedirected)
+        {
+            Assert.Throws<IOException>(() => Console.WindowWidth = 0);
+            Assert.Throws<IOException>(() => Console.WindowHeight = 0);
+        }
+        else
+        {
+            Assert.Throws<ArgumentOutOfRangeException>("width", () => Console.WindowWidth = 0);
```

Comment:
Does the ConsolePal.Window* also check arguments? If not how would you get the ArgumentOutOfRangeException now on .NET Core?